test/development/acceptance-app/ReactRefreshLogBox.test.ts

```diff
@@ -1446,6 +1575,12 @@ describe('ReactRefreshLogBox app', () => {
          "stack": [],
        }
       `)
+    } else if (isRspack) {
```

Comment:
```suggestion
    } else if (isRspack) {
      // rspack returns error content that contains absolute paths which are non deterministic
      // using toInclude instead of inline snapshot to check redbox content for correctness
```